Build.PL

```diff
@@ -24,6 +24,7 @@ my $builder = Module::Build->new(
         'JSON::XS'           => 0,
         'MIME::Lite'         => 0,
         'Net::SMTP::SSL'     => 0,
+        'Text::CSV'          => 0,

```

Comment:
Ah, you mentioned putting it in build_requires.  Now it all makes sense to me.  

I presume it would also be placed in recommends to indicate it's optional after installation, too.